cypress/support/step_definition/course/commonCourseSteps.spec.js

```diff
@@ -1,11 +1,22 @@
-const { When, Then } = require("@badeball/cypress-cucumber-preprocessor");
-import Management from '../../pages/admin/pageAdministration';
+const { When, Then, Given } = require("@badeball/cypress-cucumber-preprocessor");
+import Management from "../../pages/admin/pageAdministration";
 import Courses from "../../pages/course/pageCourses";
-import CourseManagement from '../../pages/course_management/pageCourseManagement';
+import CourseManagement from "../../pages/course_management/pageCourseManagement";
 
 const courses = new Courses();
 const management = new Management();
 
+Given("a course with name {string} exists with {string} as teacher and {string} as student", (courseName, teacherName, studentName) => {
+	courses.navigateToCoursesOverview();
+	courses.clickOnCreateCourseFAB();
+	courses.fillCourseCreationForm(courseName);
+	courses.selectCourseColour();
+	courses.selectTeacherInCourseCreatePage(teacherName);
+	courses.clickOnNextStepsBtnAfterEnteringCourseDetails();
+	courses.selectStudentInCourseCreatePage(studentName);
+	courses.clickOnNextStepButtonOnCourseParticipationDetail();
```

Comment:
add after this method some assertion as sometimes test fails as course was not created and cypress moved to next step, also check for this selector as there is a typo `#nextButtonToCreateCourseOnParticipationDeatilStep`

<img width="1726" alt="Screenshot 2024-12-11 at 12 30 48" src="https://github.com/user-attachments/assets/3bef7fbc-7def-4fbe-ab01-4d3539582358">